pkg/cluster/sync.go

```diff
@@ -612,6 +612,9 @@ func (c *Cluster) syncDatabases() error {
 	if err := c.initDbConn(); err != nil {
 		return fmt.Errorf("could not init database connection")
 	}
+
+	// only try to close the last connection that will be initialized, as c.initDbConn/c.initDbConnWithName
+	// try to close the existing connection in c.pgDb before initializing a new one
```

Comment:
Can we remove this, this does not do anything and imho the comment is also not correct, the defer part comes post init.

Reply:
`the defer part comes post init` I didn't get it?
I was trying to say that, in case we enter in the loop in line 666, the first `initDbConnWithName` in line 667 will try to close the connection we open in line 612 and that this defer will close the last connection that we open in line 667 so we don't need to add another close.
I'll remove it.

Reply:
maybe remove it if it confuses Jan ;)
I think from the code it should be clear that the last connection opened in this method will be closed in the defer part.

Reply:
yep, and to underline the fact that initDbConnWithName (tries to) does close the conn before creating another one so we don't risk a conn risk.
Done.